src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -593,6 +593,16 @@ public String getStatus() {
     }
 
     /**
+     * Returns the contents of the href attribute on an link in a table cell.
+     */
+    public String getCellLinkHrefFromDataTable(int row, int column) {
+        WebElement tableElement = browser.driver.findElements(By.className("table")).get(0);
```

Comment:
Follow the more general approach of `getCellValueFromDataTable` (include a table index instead of hardcoding `0`).